dev/5_load_environment.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+set -euxo pipefail
+
+. utils.sh
+
+function main() {
+  ./teardown_resources.sh
+
+  set_namespace "$APP_NAMESPACE_NAME"
+
+  docker_login
+
+  configure_secret
+
+  deploy_env
+
+  verify_secret_in_env
+}
+
+function docker_login() {
+  announce "Creating image pull secret."
+
+  if [[ "${PLATFORM}" == "kubernetes" ]]; then
+       $cli_with_timeout delete --ignore-not-found secret dockerpullsecret
+
+       $cli_with_timeout create secret docker-registry dockerpullsecret \
+        --docker-server=$DOCKER_REGISTRY_URL \
+        --docker-username=_ \
+        --docker-password=_ \
+        --docker-email=_
+  elif [[ "$PLATFORM" == "openshift" ]]; then
+      $cli_with_timeout delete --ignore-not-found secrets dockerpullsecret
+
+      # TODO: replace the following with `$cli create secret`
+      $cli_with_timeout secrets new-dockercfg dockerpullsecret \
+            --docker-server=${DOCKER_REGISTRY_PATH} \
+            --docker-username=_ \
+            --docker-password=$($cli_with_timeout whoami -t) \
+            --docker-email=_
+
+      $cli_with_timeout secrets add serviceaccount/default secrets/dockerpullsecret --for=pull
+  fi
+}
+
+function configure_secret() {
+  announce "Configuring K8s Secret and access."
+
+  export ENV_DIR="$PWD/config/k8s"
+  if [[ "$PLATFORM" = "openshift" ]]; then
+      export ENV_DIR="$PWD/config/openshift"
+  fi
+
+  echo "Create secret k8s-secret"
+  $cli_with_timeout create -f $ENV_DIR/k8s-secret.yml
+
+  create_secret_access_role
+
+  create_secret_access_role_binding
+
+  set_namespace $CONJUR_NAMESPACE_NAME
+
+  set_secret "secrets/test_secret" "supersecret"
+}
+
+function verify_secret_in_env() {
+  announce "Verifying that K8s Secret is in Application container"
+
+  # TODO change test-env
+  echo "Verifying pod test_env has environment variable 'TEST_SECRET' with value 'supersecret'"
```

Comment:
@eladkug you mean an env var correct? not a conjur variable

Reply:
i agree. it will make it easier to test other types of conjur secrets. for example test that we can retrieve secrets with special characters so i need to change the secret value only in one place